src/promptflow-core/promptflow/executor/_service/utils/process_utils.py

```diff
@@ -81,15 +82,19 @@ async def invoke_sync_function_in_process(
                 ProcessManager().remove_process(run_id)
 
 
-def _execute_target_function(
+def _set_environment_variables_then_execute_target_function(
```

Comment:
Maybe there is no need to change the function name, just adding a parameter would be enough.